docs/ethereum/overview.md

```diff
@@ -5,7 +5,10 @@ slug: /ethereum
 
 # Overview
 
-Brave injects a `windows.ethereum` provider object on all pages.  
+Brave injects a `windows.ethereum` provider object on all secure sites.
+Secure 1st party iframes will also have `window.ethereum` exposed but will work the same as other secure feature policies.
+For more information please see [this work](https://github.com/brave/brave-browser/issues/23710).
+
```

Comment:
This also needs to be added into Solana overview section. Or we can just move it to `provider-objects.md`